packages/navbar/src/NavDropdown.js

```diff
@@ -95,13 +96,20 @@ export default class NavDropdown extends Component {
             activeItem: item
           })
         }}
+        id={name.length > 10 && `custom-nav-${index}`}
```

Comment:
it's confused me, why the `id` property bind with `name.length`

Reply:
Because only those name length exceeds the limit need to add the attribute id